.github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md

```diff
@@ -0,0 +1,85 @@
+
+PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved. 
+
+Expedition, questions, and discussion can happen at any meeting.
+
+Delays in review & merging should be addressed in meetings or with NMDC leadership.
+
+# PR Information
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation
+- [ ] Schema change: Structure and content
+  - created, updated, or deleted a `class`, `slot`, or `enum`
+  - changed whether a `slot` is `multivalued`
+  - changed the way a `slot` is assigned to a `class`
+  - changed the `permissible_values` of an `enum`
+  - _etc._
+- [ ] Schema change: Cleanup and preparation
+  - updated the description of a `class`, `slot`, or `enum`
+  - updated the `mappings` of a `class`, `slot`, or `enum` to an ontology
+  - added an `enum` for _future_ use (it is not in the `range` of any `slot`)
+  - _etc._
+     
+## Description
+
+> _PRs should be [small and concise](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#write-small-prs)._
+>
+> Aim to create small, focused pull requests that fulfill a single purpose. Smaller pull requests are easier and faster to review and merge, leave less room to introduce bugs, and provide a clearer history of changes.
+
+- Replace this text with a description of what this PR branch contains. Please keep in mind that all reviewers will be reading this description. Example: "In this branch, I..."
+
+## Related Issues
+
+> All PRs should relate to or fix an issue(s). Please identify the issue(s) below.
+
+- Related Issue(s): #
+- Fixes: #
+
+## Did you add/update any tests?
+
+- [ ] Yes
+- [ ] No _(Add a justification below)_
+- [ ] I need help with writing tests
+
+## Could this schema change make it so any valid data becomes invalid?
+
+> This is a question about what the schema allows. It is not a question about what happens to exists in the NMDC database right now.
+> 
+> Example: If, in this PR branch, you renamed a `slot` from `foo` to `foo_bar`, the answer to this question would be "yes," **even if** nothing in the NMDC database _currently_ uses the `foo` `slot`.
+>
+> More examples: `slot` or `class` name changes, changes to a `slot`'s `multivalued` state, changes to a `slot`'s `range` (e.g. string to integer), changes to `slot` assignments to `class`es, changes to an `enum`'s `permissible_values`
+
+- [ ] Yes _(A migrator is required)_
+- [ ] No
+- [ ] I need help determining this
+
+## If you answered "Yes", does this PR branch include that migrator?
+
+- [ ] Yes
+- [ ] No, this PR is incomplete and I need help writing the migrator
+
+## Does this PR have any downstream implications?
+
+> **Ideally**, no schema changes will be performed that have downstream implications.
```

Comment:
I propose removing this line as I think it's redundant with the intro. Also, phrasing of a "schema change" being "performed" (as opposed to "made" or "applied") doesn't sound natural to my ears.